api/src/main/java/com/debugmafia/clueless/model/Player.java

```diff
@@ -0,0 +1,52 @@
+package com.debugmafia.clueless.model;
+
+import java.util.Set;
+
+public class Player {
+
+  private String userName;
+  private int playerNum;
+  private Piece character;
+  private boolean isInControl;
```

Comment:
Don't think we need the `isInControl` flag - the `Game` instance knows this information only.